src/sider/oppfolgingsplan/oppfolgingsplaner/Oppfolgingsplan.tsx

```diff
@@ -23,16 +36,12 @@ const PlanVisning = ({ dokumentinfo, oppfolgingsplan }: PlanVisningProps) => {
     }
   }
 
-  const TilbakeTilOppfolgingsplaner = () => {
-    return (
-      <div className="blokk">
-        <Link to={`/sykefravaer/oppfoelgingsplaner`} className="tilbakelenke">
-          Til oppfølgingsplaner
-        </Link>
-      </div>
-    );
-  };
-
+  if (isLoading) {
+    return <AppSpinner />;
+  }
+  if (isError) {
+    return <Feilmelding />;
+  }
```

Comment:
Hadde vært "kult" å ha en wrapper-komponent som håndterte sånn her rundt omkring i appene våre. Vi har også loading/spinners på et veldig høyt nivå. Men det er en annen diskusjon, altså ☺️

Reply:
Ja, dette tror jeg vi kunne hatt veldig nytte av å rydde opp i 😁 Så hadde vi sluppet alle disse `…Container` komponentene som vi har liggende rundt `…Side` komponenter.

Reply:
https://react.dev/reference/react/Suspense kunne jo løst noe av dette, men det er nok en større endring i hvordan vi laster data til komponentene også :thinking: Tipper evt modernisering av frontendene som vi har snakket om også vil åpne muligheter for å gjøre dette bedre.